src/table/src/engine.rs

```diff
@@ -59,6 +61,19 @@ impl<'a> Display for TableReference<'a> {
     }
 }
 
+/// CloseTableResult
+///
+/// Returns [`CloseTableResult::Released`] and closed region numbers if a table was removed
+/// from the engine.
+/// Returns [`CloseTableResult::PartialClosed`] and closed region numbers if only partial
+/// regions were closed.
+#[derive(Debug, PartialEq)]
```

Comment:
Is deriving `PartialEq` really needed? Is `CloseTableResult::Released(vec[1,2])` equals to `CloseTableResult::Released(vec[2,1])`?